docs/openapi/api-internal-pn-bff.yaml

```diff
@@ -17,16 +17,23 @@ servers:
 paths:
   '/status':
     get:
-      summary: healthCheck path
-      description: health check path per verificare lo stato del micro servizio
+      summary: Get Status of application
+      operationId: getStatus
       tags:
-        - HealthCheck
-      operationId: status
+        - StatusTemplate
       responses:
         '200':
-          description: Ok
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: string
         '500':
-          description: Internal Server Error
+          description: InternalServerError
+          content:
+            application/problem+json:
+              schema:
+                $ref: 'remote-refs.yaml#/components/schemas/Problem'
   '/template/api/httpheaders':
```

Comment:
As before

Reply:
I think to handle errors. It's the same on pn-delivery-push